apis/logging/v1/filter_api_audit_types.go

```diff
@@ -0,0 +1,69 @@
+package v1
+
+import auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
+
+// APIAudit filter Kube API server audit logs, as described in [Kubernetes Auditing].
+//
+// # Policy Filtering
+//
+// Policy event rules are the same format as the [Kube Audit Policy] with some minor extensions.
+// The extensions are described here, see the [Kube Audit Policy] for the standard rule behavior.
+// Rules are checked in order, checking stops at the first matching rule.
+//
+// The `level` of the rule determines how the event is treated:
+//   - None: the event is dropped.
+//   - Metadata: Audit metadata only, request and response objects are removed.
+//   - Request: Audit metadata and request are retained, response is removed.
+//   - RequestResponse: All data is retained, including the full response. This can lead to very large events.
```

Comment:
Should this be: Full, Complete, All, Entire since its everything?

Reply:
Which bit? It says: _All_ data is retained, including the _full_ response

Reply:
I'm suggesting the enum type should maybe be one of: Full, Complete, All, Entire

Reply:
Gotcha -  the enum values come from the k8s audit policy spec so changing them would be confusing. I agree RequestResponse is a bit odd. It does emphasize that you will be including the response body, which is what really blows up the audit records.